catalogs/sen1floods11-mldata/main.py

```diff
@@ -19,12 +20,18 @@ def mapper(item):
     from multiple image scenes.
 
     """
+    global S1
     source_links = list(filter(lambda l: l.rel == "source", item.links))
     for link in source_links:
         link.resolve_stac_object()
     source_items = [link.target.clone() for link in source_links if "_S1" in link.target.id]
     if len(source_items) == 0:
         print("WARNING: No source images for {}".format(item.id))
```

Comment:
The raw sen1floods11 STAC matches weak Sentinel-2 labels with the corresponding (Sentinel-2) source imagery that created them.   That makes sense, but  here we need to train and predict on Sentinel-1 imagery.